MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -3194,13 +3194,13 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 	setMemoryType(typeIdx, sharedHeapIdx, sharedTypePropFlags);
 	typeIdx++;
 
-	// Managed storage
+	// Managed storage. On all Apple Silicon, use Shared instead.
 	uint32_t managedBit = 0;
-#if MVK_MACOS
```

Comment:
Another potential hunk of dead code on iOS/tvOS...

Reply:
Fixed.